ERP/routes/web.php

```diff
@@ -121,3 +122,9 @@
     Route::get('update-bike/{id}', [BikeController::class, 'updateBike'])
         ->name('update.bike');
 });
+
+//Shipping Routes given `shipping.access.only` middleware (prevents non-shipping or non-IT personal from accessing this route)
```

Comment:
IT personal can access the Shipping page?

Reply:
What's the super-user now?

Reply:
We technically don't have a super user yet

Reply:
ok i will take it out for you.